academic-services/graph/network-similarity-databricks-getsimilarity.md

```diff
@@ -4,38 +4,37 @@ description: NetworkSimilarity.getSimilarity Function
 services: microsoft-academic-services
 ms.topic: extra
 ms.service: microsoft-academic-services
-ms.date: 9/23/2020
+ms.date: 4/16/2021
 ---
 # NetworkSimilarity.getSimilarity Function
 
-### Compute the similarity score between two entities.
+## Compute the similarity score between two entities.
 
 ## float getSimilarity(entityId1, entityId2)
```

Comment:
Suggestion: Add a ## header named "Signature" or something similar and then have this as a code block underneath it